src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -153,6 +153,8 @@ def check_can_update(name: UUID) -> Any:
 
 def reimage_vmss_nodes(name: UUID, vm_ids: List[UUID]) -> Optional[Error]:
```

Comment:
I'm assuming you already checked, but just in  case: are there are any other places where we pass lists of IDs (VM or otherwise) to Azure, where we should defensively dedupe / pass around sets?

Reply:
Yep, I checked.